src/components/icon/_star.less

```diff
@@ -9,10 +14,11 @@
     font-variant: normal;
     text-rendering: auto;
     -webkit-font-smoothing: antialiased;
+    transition: color 0.25s;
```

Comment:
Remove transition as it is making the button look bad. Don't forget to run `npm run compile` afterwards

Reply:
Got it.